src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -510,26 +515,16 @@ struct VSOutputs {
     }
   }
 
-  runTest(
-    buffers: VertexState<
-      {
-        stepMode: GPUInputStepMode;
-        arrayStride: number;
-        vbOffset?: number;
-      },
```

Comment:
note: you can get these to flow on one line by replacing `{\n` with `{` and then reformatting. I did this in the definition of VertexLayoutState.

Reply:
Ah, thanks. It's a bit unfortunate that npm fix has state like that but I'll make sure to test that next time.